contracts/programs/keystone-forwarder/src/events.rs

```diff
@@ -0,0 +1,33 @@
+use anchor_lang::prelude::*;
+
+#[event]
+pub struct ReportProcessedEvent {
```

Comment:
nit: remove the `Event` suffix. This isn't a common convention for Solana contracts